rocAL/include/readers/image/image_reader.h

```diff
@@ -146,6 +148,25 @@ struct ImageRecordIOHeader {
                            */
 };
 
+struct NumpyHeaderData {
+   public:
+    std::vector<unsigned> _shape;
+    RocalTensorDataType _type_info;
+    bool _fortran_order = false;
+    int64_t _data_offset = 0;
```

Comment:
Why are public variables having _ as prefix?
Only private variables should have the _ in beginning